third_party/2and3/pymysql/cursors.pyi

```diff
@@ -17,34 +18,35 @@ class Cursor:
     def __init__(self, connection: Connection) -> None: ...
     def __del__(self) -> None: ...
     def close(self) -> None: ...
-    def setinputsizes(self, *args): ...
-    def setoutputsizes(self, *args): ...
-    def nextset(self): ...
-    def execute(self, query: str, args: Optional[Any] = ...) -> int: ...
-    def executemany(self, query: str, args) -> int: ...
-    def callproc(self, procname, args=...): ...
+    def setinputsizes(self, *args) -> None: ...
+    def setoutputsizes(self, *args) -> None: ...
+    def nextset(self) -> Optional[bool]: ...
+    def mogrify(self, query: Text, args: QueryArgs = ...) -> Text: ...
+    def execute(self, query: Text, args: QueryArgs = ...) -> int: ...
+    def executemany(self, query: Text, args: Iterable[QueryArgs]) -> int: ...
```

Comment:
This can return `None` if `args` is falsy.

Reply:
Added `Optional`, thanks